### DISCUSSION TITLE
CSHTML For Javascript Extraction

### DISCUSSION THREAD

**Discussion by mwest67** [github/securitylab#305] [Q&A] [answered]
Is there any way to get the javascript extractor to consider .cshtml files for javascript extraction? The docs dont seem to suggest so

**Comment by asgerf** [comment, marked as answer]
There is not yet a standardized way to do this with the `codeql` CLI or code scanning, but as a workaround you can set the following environment variable:
```sh
LGTM_INDEX_FILETYPES=.cshtml:HTML
```
